modules/tinymce/src/plugins/lists/main/ts/core/Delete.ts

```diff
@@ -260,6 +263,13 @@ const backspaceDelete = (editor: Editor, isForward: boolean): boolean => {
 };
 
 const setup = (editor: Editor): void => {
+  editor.on('ExecCommand', (e) => {
+    const cmd = e.command.toLocaleLowerCase();
```

Comment:
```suggestion
    const cmd = e.command.toLowerCase();
```
We probably don't want this differing based on the users locale.

Reply:
Ooops. Good catch I should have seen that.